projects/angular-cc-library/package.json

```diff
@@ -23,7 +23,7 @@
     "tslib": "^2.8.1"
   },
   "peerDependencies": {
-    "@angular/common": "^19.1.4",
-    "@angular/core": "^19.1.4"
+    "@angular/common": "^20.3.13",
+    "@angular/core": "^20.3.13"
```

Comment:
Does the compiled code still compatible with a 19 version? If yes - then constraint here should be widened not just bumped to 20. 

eq: `"@angular/common": "^19 || ^20",`

Also there is no sense to narrow constraint to `20.3.13`, it should work well with every 20.x version right?

Reply:
I've changed it now. I tested by changing the demo app's version to 19, running tests, then smoke testing the app itself manually. It seems to work. 

Although I do want to note that the peer deps before for \@angular/common and \@angular/core were both ^19.1.4. I've changed it to ^19.0.0 || ^20.0.0. Meaning it's even less restrictive than the previous version.

To me that seems fine. Let me know if I should revert back to the more restrictive "^19.1.4".